config/crd/bases/_.yaml

```diff
@@ -0,0 +1,21 @@
+
```

Comment:
What is the purpose of this file?

Reply:
it's an issue of the operator sdk. I'm am chatting with the guys who are supposed to help me fix it. not sure what to say it gets generated ...

Reply:
If it gets generated, please include in a `.gitignore` file

Reply:
done